query-node/manifest.yml

```diff
@@ -0,0 +1,55 @@
+version: '0.1'
+dataSource:
+  kind: substrate 
+  chain: joystream
+  indexerVersion: '0.1.6'
+metadata:
```

Comment:
This manifest file looks like it was made for live network? does it need to be changed to work for local development testing?

Reply:
The most important part is the metadata section which has `source` and `blockHash`, for local testing one should update them.